src/cursor/cur_oligarch.c

```diff
@@ -1170,38 +1170,62 @@ __coligarch_reserve(WT_CURSOR *cursor)
 static int
 __coligarch_largest_key(WT_CURSOR *cursor)
```

Comment:
Just curious, what was the issue with the generic implementation that just called `prev` on the oligarch cursor? If it didn't work, it probably means that we have a bug in `prev`?

Reply:
It need to set the WT_CURSTD_KEY_ONLY on the ingest table and stable table's cursor not on the high level oligarch cursor. However, internally the coligarch_prev function may open and close those cursors so setting the flags in the largest_key function doesn't work.